### ISSUE TITLE
The images in train and test csv files are 40063, which is different from 47000 images in paper?

### ISSUE THREAD

**Issue by zhangyuxuann** [clobotics/piou#6]
Are there val.csv? By the way, the paper pointed out that "we selected half of the original images as the training set, 1/6 as validation set, and 1/3 as the testing set"

**Comment by zhangyuxuann** [comment]
另外能否解释一下标注？同一张图，第二个字段有些是一个框，有些是多个框，是出于什么样的考虑？ ‘1000050’表示什么？归一化的标注是对应于把原图reshape到(600,800)之后的标注是吗？
1128186,https://fileman.clobotics.cn/api/file/428711869178469a59ae2d50013a4fc3,"[[0.004991853572295213,0.65368509212730319],[0.87690227753319239,0.6335845896147404],[0.87690227753319239,0.69221105527638194],[0.0027732519846084516,0.708961474036851]]",1,[],0
1128186,https://fileman.clobotics.cn/api/file/428711869178469a59ae2d50013a4fc3,"[[0.88799528547162621,0.017169179229480736],[0.88133948070856594,0.30025125628140703],[0.90130689499774674,0.30025125628140703],[0.90574409817312029,0.37562814070351758],[0.87246507435781884,0.37395309882747069],[0.87246507435781884,0.62688442211055273],[0.88799528547162621,0.62688442211055273],[0.88577668388393938,0.69723618090452266],[0.86359066800707174,0.6989112227805695],[0.87912087912087911,0.99371859296482412],[0.99670676326827745,0.9970686767169179],[0.99448816168059073,0.0020938023450586263]]",**1000050**,[],0
1128188,https://fileman.clobotics.cn/api/file/9181382dd4786737766c66d5ae9a4cdf,"[[0.11892797319932999,0.35050251256281406],[0.983249581239531,0.35217755443886095],[0.98548297040759347,0.38735343383584592],[0.11669458403126745,0.3806532663316583]]",1,[],0

**Comment by zhangyuxuann** [comment]
It also seems that there are 2987 images in retail50k_train_1.csv that are duplicated in retail50k_train_2.csv?

**Comment by zhangyuxuann** [comment]
I don’t know if there is any problem with my parsing your .csv data. I feel that there are a lot of missing and mislabeled labels in the annotations.  Here are some visual examples http://10.224.29.12:8081/showdirs?dirs=Retail50k
